content/language/r/develop.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Use containers for R development
+keywords: R, local, development
+description: Learn how to develop your R application locally.
+---
+
+## Prerequisites
+
+Complete [Containerize a R application](containerize.md).
+
+## Overview
+
+In this section, you'll learn how to set up a development environment for your containerized application. This includes:
+
+- Adding a local database and persisting data
+- Configuring Compose to automatically update your running Compose services as you edit and save your code
+
+## Get the sample application
+
+You'll need to clone a new repository to get a sample application that includes logic to connect to the database.
+
+Change to a directory where you want to clone the repository and run the following command.
+
+```console
+$ git clone https://github.com/mfranzon/r-docker-dev.git
+```
+
+
+## Add a local database and persist data
+
+You can use containers to set up local services, like a database. In this section, you'll update the `compose.yaml` file to define a database service and a volume to persist data.
+
+In the cloned repository's directory, open the `compose.yaml` file in an IDE or text editor. 
+
+In the `compose.yaml` file, you need to uncomment all of the database instructions. In addition, you need to add the database password file as an environment variable to the server service and specify the secret file to use .
+
+The following is the updated `compose.yaml` file.
```

Comment:
The db lines weren't commented out in the compose.yaml in the repository

Reply:
I commented the lines in the `compose.yaml`